python/dask_cudf/dask_cudf/tests/test_accessor.py

```diff
@@ -438,3 +438,88 @@ def test_sorting(data, ascending, na_position, ignore_index):
         .reset_index(drop=True)
     )
     assert_eq(expect, got)
+
+
+#############################################################################
+#                            Struct Accessor                                #
+#############################################################################
+
+
+@pytest.mark.parametrize(
+    "data",
+    [
+        [{"a": 5, "b": 10}, {"a": 3, "b": 7}, {"a": -3, "b": 11}],
+        [{"a": None, "b": 1}, {"a": None, "b": 0}, {"a": -3, "b": None}],
+        [{"a": 1, "b": 2}],
+        [{"b": 3, "c": 4}],
+    ],
```

Comment:
It looks like we are repeating these same parameters in every new test.  I do like that the tests are all simple and distinct. So, perhaps we should just define this list of parameters  as `struct_accessor_data_params` above this test (and use the same variable in `@pytest.mark.parametrize` for every new test?

Reply:
Thanks, Rick!  I implemented these suggested changes and have just pushed them!